### PR TITLE
Add some indonesian source's

### LINKED ISSUE #10583: Manhwalist
### Source name

Irreguler Manhwa

### Source link

https://manhwalist.com

### Language

Indonesian

### Other details

Please add to WPmangaStream

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #11865: Ryukonesia
### Source name

Ryukonesia

### Source link

https://ryukonesia.net/

### Language

Indonesia 

### Other details

Screenshot_2022-05-14-23-03-33-183_com UCMobile intl

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #12524: manhwadesu
### Source name

manhwadesu

### Source link

https://manhwadesu.me

### Language

Indonesia

### Other details

Nsfw 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #12621: Manhwaland.mom
### Source name

Manhwaland

### Source link

https://manhwaland.mom/

### Language

Indonesian 

### Other details

😐aaaa hope it's added 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #12666: Mangayaro
### Source name

Mangayaro 

### Source link

https://mangayaro.com/

### Language

Indonesian

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #12706: Nekomik
### Source name

Nekomik

### Source link

https://nekomik.com/

### Language

Indonesia 

### Other details

Is NSFW? Uhh i think no
Since there are indeed some comics that have the Ecchi genre, I hope it doesn't count against the NSFW genre 😅

Web theme : Uses WPMangaStream theme 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #12939: LianScans
### Source name

LianScans

### Source link

https://www.lianscans.my.id

### Language

Indonesia 

### Other details

Is NSFW? True
Web Theme : Uses ~WPMangaReader~ MangaThemesia theme
Has Projects Page? True

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #13013: Dojing.net
### Source name

Dojing

### Source link

https://dojing.net

### Language

Indonesia 

### Other details

Is NSFW? True
Web theme : Uses ~WPMangaReader~ MangaThemesia theme
Has latest update? True
Has projects page? I think no

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by TheKingTermux**
Detail:

- Add 8 New Source to MangaThemesia

Closed #13013
Closed #12939
Closed #12666 
Closed #12524
Closed #12706
Closed #11865
Closed #10583
Closed #12621

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/wpmangastream/manhwalist/src/ManhwaList.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import eu.kanade.tachiyomi.lib.ratelimit.RateLimitInterceptor
+import eu.kanade.tachiyomi.multisrc.wpmangastream.WPMangaStream
+import okhttp3.OkHttpClient
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class ManhwaList : WPMangaStream("ManhwaList", "https://manhwalist.com", "id", SimpleDateFormat("MMMM dd, yyyy", Locale.forLanguageTag("id"))) {
+    private val rateLimitInterceptor = RateLimitInterceptor(4)
```
Comment (alessandrojean) [inline multisrc/overrides/wpmangastream/manhwalist/src/ManhwaList.kt:11]: Use the provided `.rateLimit(4)` directly in the client builder instead.

File: multisrc/overrides/wpmangastream/nekomik/src/Nekomik.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import eu.kanade.tachiyomi.lib.ratelimit.RateLimitInterceptor
+import eu.kanade.tachiyomi.multisrc.wpmangastream.WPMangaStream
+import okhttp3.OkHttpClient
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class Nekomik : WPMangaStream("Nekomik", "https://nekomik.com", "id", SimpleDateFormat("MMMM dd, yyyy", Locale.forLanguageTag("id"))) {
+    private val rateLimitInterceptor = RateLimitInterceptor(4)
```
Comment (alessandrojean) [inline multisrc/overrides/wpmangastream/nekomik/src/Nekomik.kt:11]: Same here.

File: multisrc/overrides/wpmangastream/ryukonesia/src/Ryukonesia.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import eu.kanade.tachiyomi.lib.ratelimit.RateLimitInterceptor
+import eu.kanade.tachiyomi.multisrc.wpmangastream.WPMangaStream
+import okhttp3.OkHttpClient
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class Ryukonesia : WPMangaStream("Ryukonesia", "https://ryukonesia.net", "id", SimpleDateFormat("MMMM dd, yyyy", Locale.forLanguageTag("id"))) {
+    private val rateLimitInterceptor = RateLimitInterceptor(4)
```
Comment (alessandrojean) [inline multisrc/overrides/wpmangastream/ryukonesia/src/Ryukonesia.kt:11]: And here.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: multisrc/overrides/wpmangastream/manhwalist/src/ManhwaList.kt (hunk 1 above)
Comment (TheKingTermux) [reply multisrc/overrides/wpmangastream/manhwalist/src/ManhwaList.kt:11]: Okay
I'm working on it

File: multisrc/overrides/wpmangastream/nekomik/src/Nekomik.kt (hunk 2 above)
Comment (TheKingTermux) [reply multisrc/overrides/wpmangastream/nekomik/src/Nekomik.kt:11]: Okay 👍🏻

File: multisrc/overrides/wpmangastream/ryukonesia/src/Ryukonesia.kt (hunk 3 above)
Comment (TheKingTermux) [reply multisrc/overrides/wpmangastream/ryukonesia/src/Ryukonesia.kt:11]: Okay 👍🏻

**Comment by TheKingTermux** [comment]
Wait a minute, haven't WPMangaStream and WPMangaReader been combined into one in MangaThemesia theme?
But why does the WPMagaStream folder reappear?
Even though I deleted it 🤔

**Comment by stevenyomi** [comment]
You really need to learn how to use [Git](https://git-scm.com/) properly.

**Comment by TheKingTermux** [comment]
> You really need to learn how to use [Git](https://git-scm.com/) properly.

Yeah
I think so

**Review by alessandrojean — APPROVED** [review]

**Pull request merged**